source/Preferences.cpp

```diff
@@ -37,7 +37,10 @@ namespace {
 	const string EXPEND_AMMO = "Escorts expend ammo";
 	const string FRUGAL_ESCORTS = "Escorts use ammo frugally";
 
-	const vector<double> ZOOMS = {.25, .35, .50, .70, 1.00, 1.40, 2.00};
+	// This controls the range of zoom scales the player can switch to.
+	// larger values are close-up and small values are farther away
+	const vector<double> ZOOMS = {.1, .2, .3, .4, .5, .6, .7, .8, .9, 1.0,
+		1.1, 1.2, 1.3, 1.4, 1.5, 1.6, 1.7, 1.8, 1.9, 2.0, 3.0, 4.0};
 	int zoomIndex = 4;
```

Comment:
> Previously the zoom progressed erratically: .25, .35, .5, .7, .1, .1.4, 2

The previous scale was not actually erratic at all. It went up in factors of approximately the square root of two with each step.
In game, this means that moving by two steps results in the dimensions doubling (or halving).

Your version, on the other hand, will appear very erratic because the factor by which the zoom level changes with each step also changes with each step, and whatever pattern is there is copmletely discarded at the very highest zoom levels.

```suggestion
	const vector<double> ZOOMS = {.125, 1.75, .25, .35, .5, .7, 1., 1.4, 2., 2.8, 4.};
	int zoomIndex = 6;
```

There's also the issue that existing preference files will now not refer to the same zoom value. That can be remedied by putting all new valuse at the end of the vector.

```suggestion
	const vector<double> ZOOMS = {.25, .35, .5, .7, 1., 1.4, 2., 2.8, 4., .125, .175};
	int zoomIndex = 4;
```